docs/pages/includes/node-logins.mdx

```diff
@@ -0,0 +1,30 @@
+When Teleport's Auth Service receives a request to list Teleport Nodes (e.g., to
+display Nodes in the Web UI or via `tsh ls`), it only returns the Nodes that the
+current user is authorized to access.
+
+For each Node in the user's Teleport cluster, the Auth Service applies the
+following checks in order and, if one check fails, hides the Node from the user:
+
+{/*
+
+ TODO: We might want to mention that the Auth Service checks the resource's
```

Comment:
Namespaces are basically not used at all in Teleport. All resources always have `default` namespace and I don't think this will ever change TBH. So I think this "todo" item won't ever get resolved so we may as well just remove it. But it's up to you if you want to keep it.